pages/Getting Started/Installation.md

```diff
@@ -367,3 +367,71 @@ sudo cp ./build/Hyprland /usr/bin && sudo cp ./example/hyprland.desktop /usr/sha
 
 Lastly, copy hyprctl, hyprpm, and wlroots as mentioned
 [here](#manual-releases-linux-only)
+
+## Running In a VM
+*YMMV, this is not officially supported.*
+
+You may have tried to get Hyprland working in a libvirt VM by installing the official
+arch package and enabling virgl. In order to get this to work however, you must
+also compile from source yourself. To save you some headache I'm going to walk
+you through how I was able to get this to work. Note this guide is for an Arch
+host running an Arch guest.
```

Comment:
This reads more as a guide than a wiki. We want the text to be concise and to the point.